apigateway/templates/sign_cookies_api.json.tpl

```diff
@@ -0,0 +1,100 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "version" : "2021-09-06T09:43:11Z",
+    "title" : "${title}"
+  },
+  "basePath" : "/${environment}",
+  "schemes" : [ "https" ],
+  "paths" : {
+    "/cookies" : {
+      "get" : {
+        "produces" : [ "application/json" ],
+        "responses" : {
+          "200" : {
+            "description" : "200 response",
+            "schema" : {
+              "$ref" : "#/definitions/Empty"
+            },
+            "headers" : {
+              "Access-Control-Allow-Origin" : {
+                "type" : "string"
+              },
+              "Access-Control-Allow-Credentials" : {
+                "type" : "string"
+              }
+            }
+          }
+        },
+        "x-amazon-apigateway-integration" : {
+          "type" : "aws_proxy",
+          "httpMethod" : "POST",
+          "uri" : "arn:aws:apigateway:${region}:lambda:path/2015-03-31/functions/${lambda_arn}/invocations",
```

Comment:
Is it correct there should be a date in the uri value: `...path/2015-03-31/functions ...`

Reply:
Yeah, it seems so. That's the lambda API version, the only official page I can see it is in the [SDK documentation](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Lambda.html)